Code/Legacy/CrySystem/SystemInit.cpp

```diff
@@ -1263,6 +1263,10 @@ void CSystem::CreateSystemVars()
         0,
         "Minimum amount of time to show the level load screen. Important to prevent short loads from flashing the load screen. 0 means "
         "there is no limit.");
+#if defined(CARBONATED)
+    REGISTER_STRING("default_load_screen_config_file", "", 0, "Default Load Screen Config File.");
```

Comment:
Per my comment above, rename to "default_level_config_file"

Reply:
Renamed